jdaviz/configs/imviz/helper.py

```diff
@@ -177,14 +187,168 @@ def offset_to(self, dx, dy, skycoord_offset=False):
                     new_sky_cen = sky_cen.spherical_offsets_by(dx, dy)
                 self.center_on(new_sky_cen)
             else:
-                raise AttributeError(f'{image.label} does not have a valid WCS')
+                raise AttributeError(f'{getattr(image, "label", None)} does not have a valid WCS')
         else:
             with delay_callback(viewer.state, 'x_min', 'x_max', 'y_min', 'y_max'):
                 viewer.state.x_min += dx
                 viewer.state.y_min += dy
                 viewer.state.x_max = viewer.state.x_min + width
                 viewer.state.y_max = viewer.state.y_min + height
 
+    @property
+    def marker(self):
+        """Marker to use.
+
+        Marker can be set as follows; e.g.::
+
+            {'color': 'red', 'alpha': 1.0, 'markersize': 3}
+            {'color': '#ff0000', 'alpha': 0.5, 'markersize': 10}
+            {'color': (1, 0, 0)}
+
+        Also see: https://docs.glueviz.org/en/stable/api/glue.core.visual.VisualAttributes.html
```

Comment:
Whether it's here or in `add_markers`, maybe this should be more concrete, a la 
```suggestion
        The valid properties for markers in imviz are listed at  https://docs.glueviz.org/en/stable/api/glue.core.visual.VisualAttributes.html
```
or similar?

For astrowidgets we have to think about how this might be generalized, but I think that can be worried about later.